### PR TITLE
Feature user authentication

### PR DESCRIPTION
**Pull request by Chiranjibichapagain**
Client: google login component added
Server: Routes to login created
I can get the token in the backend!

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -19,6 +20,8 @@ import { Client } from "pg"
  */
```
Comment (Rostislav013) [inline src/app.ts:20]: Fix imports

Comment (Chiranjibichapagain) [reply src/app.ts:20]: what about it? didn't understand

Comment (Rostislav013) [reply src/app.ts:20]: Move
```
 import {
  SESSION_SECRET,
  PG_USER,
  PG_HOST,
  PG_DB,
  PG_PW,
  PG_PORT,
  PG_URI
} from "./util/secrets"
```
lower to local imports

**Comment by Rostislav013** [comment]
@MikeVedsted , shall it be merged to develop 1st?

**Comment by MikeVedsted** [comment]
> @MikeVedsted , shall it be merged to develop 1st?
Yes, please.

File: client/src/components/UserLogin/index.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        console.log(response)
+      })
+  }
+
+  const responseFailGoogle = (response: any) => {
+    console.log(response)
+  }
+
+  return (
+    <div>
```
Comment (MikeVedsted) [inline client/src/components/UserLogin/index.tsx:22]: no need for the div

**Review by MikeVedsted — CHANGES_REQUESTED** [review]
You are going to have conflicts with Rost's PR which was already merged. 
Looks like you created this branch before it was merged, or didn't pull the changes. 

Please remove these files from you PR: 
.env.example
client/.eslintcache
package-lock.json 
.vscode/settings.json (and add to .gitifnore) 

We are not using passport on this project. Please remove. 

Please rename component UserLogin to GoogleLogin, as we might implement other auth methods in the future but this method should remain separate. 
For backend, please also rename router and controller to reflect this. Your changes are related to auth, eventhough  they are specifically for google. 


The PR should be made for develop, not master.

**Comment by Chiranjibichapagain** [comment]
Wll create a new PR

**Pull request closed without merging**